docs/configuration_syntax.md

```diff
@@ -128,6 +128,18 @@ pull:
     # discovered project refs (optional, default: 30)
     interval_seconds: 30
 
+  metrics_with_traces:
```

Comment:
I think we could avoid adding another scheduler for the polling of this metric as it can be done during the fetch of the other job's metrics :thinking:

Reply:
The trace processing can take a while - depending on how long the logs are and the job count. In our particular case we want to fetch these metrics only once a day. And to keep the load on both sides (exporter and gitlab) as low as possible.

Another approach could be to trigger trace metrics only every n-th polling - but it's also another additional configuration parameter.